src/main/kotlin/app/revanced/patches/tiktok/misc/login/fixgoogle/FixGoogleLoginPatch.kt

```diff
@@ -1,24 +1,24 @@
-package app.revanced.patches.tiktok.misc.login.fixgoogle.patch
+package app.revanced.patches.tiktok.misc.login.fixgoogle
 
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
 import app.revanced.patcher.patch.BytecodePatch
-import app.revanced.patcher.patch.annotations.Patch
-import app.revanced.patches.tiktok.misc.login.fixgoogle.annotations.FixGoogleLoginCompatibility
+import app.revanced.patcher.patch.annotation.CompatiblePackage
+import app.revanced.patcher.patch.annotation.Patch
 import app.revanced.patches.tiktok.misc.login.fixgoogle.fingerprints.GoogleAuthAvailableFingerprint
 import app.revanced.patches.tiktok.misc.login.fixgoogle.fingerprints.GoogleOneTapAuthAvailableFingerprint
 
-@Patch
-@Name("Fix google login")
-@Description("Allows logging in with a Google account.")
-@FixGoogleLoginCompatibility
-class FixGoogleLoginPatch : BytecodePatch(
-    listOf(
-        GoogleOneTapAuthAvailableFingerprint,
-        GoogleAuthAvailableFingerprint
-    )
+@Patch(
+    name = "Fix google login",
```

Comment:
```suggestion
    name = "Fix Google login",
```

Capitalize proper noun.